plugins/importexport/native/filter/NativeXmlPKPPublicationFilter.inc.php

```diff
@@ -243,16 +243,12 @@ function parseAuthor($n, $publication) {
 	 * @param $n DOMElement
 	 * @param $publication PKPPublication
 	 */
-	function parseCitations($node, $publication) {
+	function parseCitations($n, $publication) {
 		$publicationId = $publication->getId();
-
-		for ($n = $node->firstChild; $n !== null; $n=$n->nextSibling) {
-			if (is_a($n, 'DOMElement')) {
-				$citationsString = $n->textContent;
-				$citationDao = DAORegistry::getDAO('CitationDAO'); /** @var $citationDao CitationDAO */
-				$citationDao->importCitations($publicationId, $citationsString);
-			}
-		}
+		$citationsString = $n->textContent;
+		$citationDao = DAORegistry::getDAO('CitationDAO');
+		$importCitations = 'importCitations';
```

Comment:
Why not just `$citationDao->importCitations`?

Reply:
My bad, I copy-pasted it from v3.1.2 as it was. Changed..